lib/autofix.js

```diff
@@ -46,7 +46,7 @@ function subscribe(options, app, elmVersion) {
  * @param {Options} options
  * @param {App} app
  * @param {ElmVersion} elmVersion
- * @returns {(data: AutofixRequest) => Promise<void>}
+ * @returns {(data: AutofixRequest) => Promise<void | never>}
```

Comment:
I'm curious, does this help TS figure out more things?

Reply:
To be explicit to developers. It also helps TS detect unused code, but not much.